src/org/zaproxy/zap/extension/domxss/TestDomXSS.java

```diff
@@ -574,5 +584,72 @@ public int getWascId() {
         		AlertThreshold.MEDIUM};
     }
 
+	@Override
+	public void scan(HttpMessage msg, NameValuePair originalParam) {
+		if (originalParam.getType() != NameValuePair.TYPE_QUERY_STRING) {
+			return; // Exit if it isn't a URL param
+		}
+		super.scan(msg, originalParam);
+	}
+
+	@Override
+	public void scan(HttpMessage msg, String paramName, String attack) {
+		Stats.incCounter("domxss.scan.count");
+
+		ArrayList<WebDriverWrapper> drivers = new ArrayList<WebDriverWrapper>();
+
+		WebDriverWrapper fxDriver;
+		try {
+			fxDriver = this.getFirefoxDriver();
+		} catch (Exception e) {
```

Comment:
You seemed to suggest (https://github.com/zaproxy/zap-extensions/pull/1403#discussion_r172333344) that this catch could go away, but it isn't handled in the other location?

Reply:
If we are in the try/finally then we already have a `WebDriverWrapper`, so `getFirefoxDriver` would return the existing one?